mappings/net/minecraft/client/InteractionManager.mapping

```diff
@@ -1,17 +1,60 @@
 CLASS net/minecraft/class_504 net/minecraft/client/InteractionManager
 	FIELD field_2104 minecraft Lnet/minecraft/client/Minecraft;
+	FIELD field_2105 noTick Z
 	METHOD <init> (Lnet/minecraft/client/Minecraft;)V
 		ARG 1 minecraft
+	METHOD method_1705 resetBlockBreaking ()V
+	METHOD method_1706 update (F)V
+	METHOD method_1707 interactBlock (IIII)V
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 direction
 	METHOD method_1708 clickSlot (IIIZLnet/minecraft/class_54;)Lnet/minecraft/class_31;
 		ARG 1 windowId
 		ARG 2 slotId
 		ARG 3 button
 		ARG 4 shift
 		ARG 5 player
+	METHOD method_1709 onContainerRemove (ILnet/minecraft/class_54;)V
```

Comment:
Container is ScreenHandler in BINY. Altho this is called purely from HandledScreen, not ScreenHandler so something like `onScreenRemoved(syncId, player)`  would be more accurate